<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyMultiSpectralForm</class>
 <widget class="GUIFormBase" name="PropertyMultiSpectralForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonGetBack">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>39</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;=</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetLibList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>161</width>
     <height>171</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidgetGeneratedLibList">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>161</width>
     <height>171</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButtonSetFromAll">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>39</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>=&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLibFolderName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ＸＸＸ</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>381</width>
     <height>331</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Phase</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Page</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>LibID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ItemID</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButtonSetFrom">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>39</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>=&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGetBackAll">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>39</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEditLibFolder">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>分類選択</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Qtres/Class.png</normaloff>:/Qtres/Class.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEditLibrary">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ライブラリ編集</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Qtres/Edit.png</normaloff>:/Qtres/Edit.png</iconset>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
